syft/file/selection.go

```diff
@@ -0,0 +1,9 @@
+package file
+
+const (
+	NoFilesSelection    Selection = "no-files"
+	OwnedFilesSelection Selection = "owned-files"
+	AllFilesSelection   Selection = "all-files"
+)
+
+type Selection string
```

Comment:
from @willmurphyscode : for the future, we could add an options for "non-owned" files... create an issue for this

Reply:
I've also incorporated the additional verbal comment about the names of these values and variables... a recent commit adjusts these